feature/signupflow/entercode/src/main/java/com/epmedu/animeal/signup/entercode/presentation/EnterCodeScreenEvent.kt

```diff
@@ -0,0 +1,5 @@
+package com.epmedu.animeal.signup.entercode.presentation
+
+internal sealed interface EnterCodeScreenEvent {
+    object ScreenDisplayed : EnterCodeScreenEvent
```

Comment:
About the naming, `ScreenDisplayed` at `HomeScreen` is used to define when screen passed the `onCreate` method. Here I guess something like "LoadingDone" or "ProcessComplete" would fit better.

Reply:
I renamed `ScreenDisplayed` at `HomeScreen` to `ScreenCreated`